spiketoolkit/sorters/kilosort/kilosort.py

```diff
@@ -85,11 +81,27 @@ def _setup_recording(self):
         source_dir = Path(__file__).parent
         
         p = self.params
+        self.kilosort_path = p['kilosort_path']
+        self.npy_matlab_path = p['npy_matlab_path']
```

Comment:
With the actual class design this will not work because if HAVE_KILOSORT = False (not in env) then this never reach this part.
This is why I delibarately remove this from that part.
My idea was to put kilosort_path and npy_matlab_path on class atribute with a class method that can change then.
In short we can't put path at the same level of other sorter params because the installed is checked before.

I will merge this but make anotehr proposal about this. Tell you when will be done.

Reply:
I see. Ok let's try to find another way to do it.
Something like KilosortSorter.set_path() sounds nice. I can make it tomorrow and update the PR if the rest looks ok. 

Can the HAVE_KILOSORT be overwritten from within the class?

Reply:
I think set_path as a classmethod that change _kilosort_path (a class attribute) the default class attribute beeing the global kilosort_path.

So we must call **KilosortSorter.set_path()** before evry thing.
 We can then change the run_kilosort signature by adding theses two path and add **set_path**
in run_kilosort.
What do you think ?
So the old signature would be OK and the path would be at class level wich make sens.
We could make the same for ironclust.

Reply:
Sounds good!